### DISCUSSION TITLE
Error while accessing events

### DISCUSSION THREAD

**Discussion by 29ayush** [kuzmoyev/google-calendar-simple-api#112] [Q&A] [answered]
While doing [ x for x in calendar.get_events( startDate, endDate, orderBy='startTime', single_events=True) ] 
I got ValueError: "entry_point" must be one of video, phone, sip, more. None was provided.
I don't know what to do.

What I want is get all calendar events which have an attachment so I was iterating over events

**Comment by kuzmoyev** [comment]
That's the issue with the events with conference data. Apparently, google can return `None` in entry_point. Will need to allow it. Thank you for finding this. Please, feel free to create an issue. I'll try to fix it ASAP.

**Comment by kuzmoyev** [comment, marked as answer]
The fix is in `v1.2.1`. Try

```
pip uninstall gcsa
pip install gcsa
```

**Comment by 29ayush** [comment]
You are amazing, thanks. 
Works now.
